### ISSUE TITLE
MD040: Create a config to allow other content

### ISSUE THREAD

**Issue by Dentrax** [DavidAnson/markdownlint#730]
https://github.com/DavidAnson/markdownlint/blob/main/doc/md040.md documentation says:
> _"Info strings with leading/trailing whitespace (ex: js ) or other content (ex: ruby startline=3) will trigger this rule."_

I want to use this rule, but I have the following:

`yaml mdox-exec="go run autogendocs.go"`

So linter says:

```
MD040/fenced-code-language Fenced code blocks should have a language specified [Info string contains more than language: "yaml mdox-exec="go run autogendocs.go""]
```

What is the workaround? Should I have to disable this?

**Comment by DavidAnson** [comment]
Per the documentation you link, info strings will only be blocked if the "language_only" parameter is set to true. That is not the default, so if you were doing so that's why it's being reported.

As you can see in the demo, your example does not warn by default: https://dlaa.me/markdownlint/#%25m%23%20Issue%20730%0A%0A%60%60%60yaml%20mdox-exec%3D%22go%20run%20autogendocs.go%22%0Avalue%3D5%0A%60%60%60%0A
